index.html

```diff
@@ -6,10 +6,70 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
+    <link rel="stylesheet" href="style.css">
 <body>
+    <header class="mainheader">
+        <div class="topheader">
+            <h3 class="topheader1">LOGO</h3>
```

Comment:
Logo will probably need to be an `img` tag not an h3

Reply:
REplaced with img tag